sklearn/metrics/_pairwise_distances_reduction/_base.pyx.tp

```diff
@@ -84,6 +83,23 @@ cdef float64_t[::1] _sqeuclidean_row_norms32_dense(
     return squared_row_norms
 
 
+cdef float64_t[::1] _sqeuclidean_row_norms64_sparse(
```

Comment:
Changing `sklearn/metrics/_pairwise_distances_reduction/_base.pyx.tp` is doing more than reverting https://github.com/scikit-learn/scikit-learn/commit/de67a4420f1713058070802ad593cbcd2ee2d5f3. 

Did you intend to make changes to `_base.pyx.tp`?

Reply:
yes, https://github.com/scikit-learn/scikit-learn/pull/25731 was based on https://github.com/scikit-learn/scikit-learn/pull/25598. Reverting the latter requires reverting the former.